CO-3329 Custom data: _merge_objects won't delete data from array of objects. (#9076)
braze-inc/braze-docs: _docs/_user_guide/data/custom_data/custom_attributes/array_of_objects.md

Update values for specific objects within an array using the `_merge_objects` parameter and the `$update` operator. Similar to updates to simple [nested custom attribute]({{site.baseurl}}/nested_custom_attribute_support/#api-request-body) objects, this performs a deep merge.

> Note that `$update` cannot be used to remove a nested property from an object inside an array. In order to do this, you will need to remove the entire item from the array, and then add the object back without that specific key (using a combination of `$remove` and `$add`).

Reviewer: ```suggestion
Note that `$update` can't be used to remove a nested property from an object inside an array. To do this, you'll need to remove the entire item from the array and then add the object without that specific key (using a combination of `$remove` and `$add`).
```
